src/diffusers/models/adapter.py

```diff
@@ -90,6 +90,8 @@ def forward(self, xs: torch.Tensor, adapter_weights: Optional[List[float]] = Non
             features = adapter(x)
             if accume_state is None:
                 accume_state = features
+                for i in range(len(accume_state)):
+                    accume_state[i] = w * accume_state[i]
```

Comment:
ok, so the fix is that the weights are not being applied for the first adapter network, nice!